cmd/kpod/spec.go

```diff
@@ -124,6 +128,22 @@ func createConfigToOCISpec(config *createConfig) (*spec.Spec, error) {
 		g.AddTmpfsMount(spliti[0], options)
 	}
 
+	for _, i := range config.env {
```

Comment:
Did you decide to not append the defaultEnvVariables if none were found?  i.e. line 34 from the deleted section of create_cli.go above.

Reply:
We are always setting the default environment above in the patch.

+       env = append(defaultEnvVariables, env...)

Reply:
I don't think the check is necessary, given that appending an empty array is a no-op

Reply:
Missed that, thx, I'll blame the cold meds.